app/handlers/sensor.handler.go

```diff
@@ -0,0 +1,175 @@
+package handlers
+
+import (
+	"database-ms/app/models"
+	sensorSrv "database-ms/app/services/sensor"
+	"database-ms/utils"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/gin-gonic/gin"
+	"gopkg.in/mgo.v2/bson"
+)
+
+type SensorHandler struct {
+	snsr sensorSrv.SensorServiceInterface
+}
+
+func NewSensorAPI(sensorService sensorSrv.SensorServiceInterface) *SensorHandler {
+	return &SensorHandler{
+		snsr: sensorService,
+	}
+}
+
+func (h *SensorHandler) Create(c *gin.Context) {
```

Comment:
We need to create a new sid that is 0-255 but does not exist for any other sensor for the thing. Is this handled?